src/diabetes/data_processor.py

```
@@ -0,0 +1,57 @@
+from pandas import read_csv
+from sklearn.preprocessing import LabelEncoder
```

Ruff

`sklearn.preprocessing.LabelEncoder` imported but unused

Remove unused import: `sklearn.preprocessing.LabelEncoder`

(F401)